tests/stage_2/curry-7.txt

```diff
@@ -1,2 +1,2 @@
-run -i clvm_runtime '((c (mod (func func_hash . args) (include curry.clvm) (include sha256tree.clvm) (curry_sha_args func func_hash args)) (list (q (+ 2 5 11)) 0x4efe22f87114f43e4292c0645b6ee567aff727a3aa8f7e5114ba395e6f2b2f7c 10 20)))'
-(((c (q (+ 2 5 11)) (c (q 10) (c (q 20) 1)))) . 0x55648d7a4b2712858f5e1b3422a55b56fc74cb76b54601fd11180549f9d12eb4)
+run -i clvm_runtime '((c (mod (func func_hash . args) (include curry.clvm) (include sha256tree.clvm) (curry_sha_args func func_hash args)) (list (q . (+ 2 5 11)) 0x4efe22f87114f43e4292c0645b6ee567aff727a3aa8f7e5114ba395e6f2b2f7c 10 20)))'
+(((c (q 12 2 5 11) (c (q . 10) (c (q . 20) 1)))) . 0xec751770ff3205f4def98456e77c70a79690c56ffd6e99c86c04e6b5ffe9ddf8)
```

Comment:
how can we know that this long binary blob is correct?

Reply:
That is a really good question. One way would be to have an alternate implementation of `sha256tree`, and run that on the in-memory program.

Reply:
Or have another test that simply unrolls the `sha256` over the expected input values that has the same blob output.